packages/devtools_app/lib/src/flutter/theme.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+ThemeData themeFor({@required bool isDarkTheme}) {
+  final theme = isDarkTheme ? _darkTheme() : _lightTheme();
+  return theme;
+}
+
+ThemeData _darkTheme() {
+  final theme = ThemeData.dark();
+  return theme.copyWith(
+      primaryColor: devtoolsGrey[900],
```

Comment:
comment on why these values where chosen. e.g. is this the standard collection of colors for a theme given the choice devtoolsGrey and devtoolsBlue?
Please include screenshots when sending CLs like this so they are easier to review.

Reply:
These are our brand colors: https://drive.google.com/open?id=1QBhMJqXyRt-CpRsHR6yw2LAfQtiNat4g

I've added screenshots.